### ISSUE TITLE
Resource pack needed

### ISSUE THREAD

**Issue by duckysuck** [Flambaa/flamba_particleeffects#1]
Are you planning to release this for fivem? Mods folders are for single player users.

**Comment by Flambaa** [comment]
I dont believe servers can stream .ypt files or .ptfx at the moment. Im pretty busy at the moment working on my own shaders. When I have some free time ill take a look and see if its possible. :)
